source/wp-content/themes/wporg-developer-blog/templates/search.html

```diff
@@ -0,0 +1,60 @@
+<!-- wp:template-part {"slug":"header"} /-->
+
+<!-- wp:wporg/local-navigation-bar {"backgroundColor":"charcoal-2","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"},":hover":{"color":{"text":"var:preset|color|white"}}}},"position":{"type":"sticky","top":"0px"}},"fontSize":"small"} -->
+	<!-- wp:site-title {"level":0,"fontSize":"small"} /-->
+
+	<!-- wp:navigation {"openSubmenusOnClick":true,"icon":"menu","overlayBackgroundColor":"charcoal-2","overlayTextColor":"white","layout":{"type":"flex","orientation":"horizontal"},"fontSize":"small","menuSlug":""} /-->
```

Comment:
This block typically has a `menuSlug` configured to load a static menu for translation. I notice the screenshot shows dropdowns included. What will happen to this block when without a `menuSlug` or a menu id?

Reply:
Yeah, it will load the first configured menu. You can actually choose the menu you want to load in the Site Editor. This menu is currently managed locally on the site in the Site Editor. In the future, it would be great to standardize what this menu contains and then include it in the theme as we do with other themes so it can be translated.